gtsfm/averaging/translation/averaging_1dsfm.py

```diff
@@ -171,46 +164,233 @@ def _get_prior_measurements_in_world_frame(
         Returns:
             BinaryMeasurementsPoint3 containing Point3 priors in world frame.
         """
-        if len(i2Ti1_priors) == 0:
-            return gtsam.BinaryMeasurementsPoint3()
 
         def get_prior_in_world_frame(i2, i2Ti1_prior):
             return wRi_list[i2].rotate(i2Ti1_prior.value.translation())
 
-        w_i2ti1_priors = gtsam.BinaryMeasurementsPoint3()
+        w_i1ti2_prior_measurements = BinaryMeasurementsPoint3()
+        if len(i2Ti1_priors) == 0:
+            return w_i1ti2_prior_measurements
 
+        # TODO(akshay-krishnan): Use the translation covariance, transform to world frame.
+        # noise_model = gtsam.noiseModel.Gaussian.Covariance(i2Ti1_prior.covariance)
+        # TODO(akshay-krishnan): use robust noise model for priors?
+        noise_model = gtsam.noiseModel.Isotropic.Sigma(3, 1e-2)
         for (i1, i2), i2Ti1_prior in i2Ti1_priors.items():
-            # TODO(akshay-krishnan): Use the translation covariance, transform to world frame.
-            # noise_model = gtsam.noiseModel.Gaussian.Covariance(i2Ti1_prior.covariance)
-            noise_model = gtsam.noiseModel.Isotropic.Sigma(3, 1e-2)
-            w_i2ti1_priors.append(
+            w_i1ti2_prior_measurements.append(
                 BinaryMeasurementPoint3(
-                    i2,
-                    i1,
+                    C(i2),
+                    C(i1),
                     get_prior_in_world_frame(i2, i2Ti1_prior),
                     noise_model,
                 )
             )
-        return w_i2ti1_priors
+        return w_i1ti2_prior_measurements
+
+    def compute_inliers(
+        self,
+        w_i2Ui1_dict: RelativeDirectionsDict,
+        w_i2Ui1_dict_tracks: RelativeDirectionsDict,
+    ) -> Tuple[RelativeDirectionsDict, RelativeDirectionsDict, Set[int]]:
+        """Perform inlier detection for the relative direction measurements.
+
+        Args:
+            w_i2Ui1_dict: Dictionary of Unit3 relative translations between cameras.
+            w_i2Ui1_dict_tracks: Dictionary of Unit3 relative translations between cameras and landmarks.
+
+        Returns:
+            Set of indices (i1, i2) which are inliers.
```

Comment:
nit: i think there should be 3 return args here for `inlier_w_i2Ui1_dict, inlier_w_i2Ui1_dict_tracks, inlier_cameras`

Reply:
Can we also clarify what `i1` and `i2` represent in this context? can be a landmark index?

Reply:
thanks, yeah, I can rename inlier_w_i2Ui1_dict_tracks to `inlier_w_cUt_dict` , but then folks have to remember that the idxs are (t, c) not (c, t). Using i2 and i1 just lets them know what the keys are. 
Also there are places (very close to the C++ interface) where these two are in the same data structure, they are combined before calling C++ code.